assets/src/web-stories-block/css/views/carousel.css

```diff
@@ -0,0 +1,78 @@
+/* Web Stories: Carousel View */
+.web-stories-list.is-view-type-carousel {
+  position: relative;
+}
+
+@media all and (min-width: 676px) {
+  .web-stories-list.is-view-type-carousel .web-stories-list__inner-wrapper {
+    max-width: var(--ws-story-max-width);
+  }
+}
+
+.web-stories-list.is-view-type-carousel .web-stories-list__story-placeholder {
+  position: relative;
+}
+
+.web-stories-list.is-view-type-carousel.is-style-default
+  .web-stories-list__story-placeholder {
+  border-radius: 8px;
+  overflow: hidden;
+}
+
+.is-view-type-carousel .glider-next,
+.is-view-type-circles .glider-next {
+  right: 0;
+  transform: translateY(-50%);
+}
+
+.is-view-type-carousel .glider-prev,
+.is-view-type-circles .glider-prev {
+  left: 0;
+  transform: translateY(-50%) rotate(180deg);
+}
+
+.is-view-type-carousel .amp-carousel-button,
+.is-view-type-carousel .glider-next,
+.is-view-type-carousel .glider-prev,
+.is-view-type-circles .amp-carousel-button,
+.is-view-type-circles .glider-next,
+.is-view-type-circles .glider-prev {
+  border-radius: 50%;
+  background-color: #eaeaea;
+  background-image: url('./../../../../images/web-stories/block/carousel-arrow.svg');
```

Comment:
I wonder if using relative URLs like this can cause issues on certain sites 🤔 
What if a caching plugin moves CSS files around?
Do we need to add this via some inline style in PHP to be safe?

@spacedmonkey WDYT?

Reply:
If you are worried about the sites where this file gets loaded then it won't be an issue as the built CSS will have base64 URI instead of the path.

Reply:
<img width="1084" alt="Screenshot 2021-01-07 at 14 34 24" src="https://user-images.githubusercontent.com/237508/103905308-5b5e0f80-50f6-11eb-9d0f-f73cad995036.png">
Confirmed by testing locally.

Reply:
That is awesome! Now I am not worried anymore :-)